src/suites/positive/positive-suite.ts

```diff
@@ -0,0 +1,60 @@
+import { OperationResult } from '../../validation';
+import { OperationExampleFactory } from '../../oas-parsing/operation-example';
+import { ValidationConductor } from './conductors';
+import Suite from '../suite';
+
+export default class PositiveSuite extends Suite {
+  public static suiteId = 'positive';
+  public static label = '(Example Group: Positive)';
+
+  async conduct(): Promise<OperationResult[]> {
+    this.checkTargetServer();
+
+    if (!this.suiteConfig.securityValues) {
+      throw new Error('Unable to run suite due to missing securityValues');
```

Comment:
The ValidationConductor expects it to be defined.  This was added to resolve TS errors.

Reply:
You could make it required on SuiteConfig instead of optional. Where it's built, `SecurityValuesFactory.buildFromSecuritySchemes()` will always return a `SecurityValues` object. It will just be an empty object if there are none